src/redux/store.js

```diff
@@ -0,0 +1,4 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from 'redux/features/books/bookSlice';
+
+export default configureStore({ reducer: { books: booksReducer } });
```

Comment:
- [ ] [REQUIRED] Kindly import `categoriesReducer` from 'redux/features/categories/categorySlice' and also add it to your reducer on line 4. This will make the category state available to your app via the configureStore.